test/integration/net_test.go

```diff
@@ -69,6 +69,10 @@ func TestNetworkPlugins(t *testing.T) {
 					t.Skipf("flannel is not yet compatible with Docker driver: iptables v1.8.3 (legacy): Couldn't load target `CNI-x': No such file or directory")
 				}
 
+				if !DockerDriver() && tc.name == "kindnet" {
```

Comment:
mind commenting why Kindnet should Not work with CNI ?

Reply:
done!